.github/workflows/check-pull-request-title.yml

```diff
@@ -0,0 +1,32 @@
+name: check-pull-request-title
+
+on:
+  pull_request:
+    types: [assigned, edited, opened, synchronize, reopened]
```

Comment:
To confirm, we need all of these PR action types, yes?

Reply:
@v-kpheng, good catch, I trimmed some because we only need two:
* `opened` for when PR is first opened, we want to check it then
* `edited` for when PR title is edited
thank you 🙏